src/libLookup/Lookup.cpp

```diff
@@ -1667,27 +2133,41 @@ bool Lookup::Execute(const vector<unsigned char>& message, unsigned int offset,
     typedef bool (Lookup::*InstructionHandler)(const vector<unsigned char>&,
                                                unsigned int, const Peer&);
 
-    InstructionHandler ins_handlers[]
-        = {&Lookup::ProcessEntireShardingStructure,
-           &Lookup::ProcessGetSeedPeersFromLookup,
-           &Lookup::ProcessSetSeedPeersFromLookup,
-           &Lookup::ProcessGetDSInfoFromSeed,
-           &Lookup::ProcessSetDSInfoFromSeed,
-           &Lookup::ProcessGetDSBlockFromSeed,
-           &Lookup::ProcessSetDSBlockFromSeed,
-           &Lookup::ProcessGetTxBlockFromSeed,
-           &Lookup::ProcessSetTxBlockFromSeed,
-           &Lookup::ProcessGetTxBodyFromSeed,
-           &Lookup::ProcessSetTxBodyFromSeed,
-           &Lookup::ProcessGetNetworkId,
-           &Lookup::ProcessGetNetworkId,
-           &Lookup::ProcessGetStateFromSeed,
-           &Lookup::ProcessSetStateFromSeed};
+    InstructionHandler ins_handlers[] = {
+        &Lookup::ProcessEntireShardingStructure,
+        &Lookup::ProcessGetSeedPeersFromLookup,
+        &Lookup::ProcessSetSeedPeersFromLookup,
+        &Lookup::ProcessGetDSInfoFromSeed,
+        &Lookup::ProcessSetDSInfoFromSeed,
+        &Lookup::ProcessGetDSBlockFromSeed,
+        &Lookup::ProcessSetDSBlockFromSeed,
+        &Lookup::ProcessGetTxBlockFromSeed,
+        &Lookup::ProcessSetTxBlockFromSeed,
+        &Lookup::ProcessGetTxBodyFromSeed,
+        &Lookup::ProcessSetTxBodyFromSeed,
+        &Lookup::ProcessGetNetworkId,
+        &Lookup::ProcessGetNetworkId,
+        &Lookup::ProcessGetStateFromSeed,
+        &Lookup::ProcessSetStateFromSeed,
+        &Lookup::ProcessSetLookupOffline,
+        &Lookup::ProcessSetLookupOnline,
+        &Lookup::ProcessGetOfflineLookups,
+        &Lookup::ProcessSetOfflineLookups,
+    };
 
     const unsigned char ins_byte = message.at(offset);
     const unsigned int ins_handlers_count
         = sizeof(ins_handlers) / sizeof(InstructionHandler);
 
+#ifdef IS_LOOKUP_NODE
+    if (ToBlockMessage(ins_byte))
+    {
+        LOG_EPOCH(INFO, to_string(m_mediator.m_currentEpochNum).c_str(),
```

Comment:
Log epoch will log the epoch. There is no need for to_string(m_mediator.m_currentEpochNum).c_str()